src/views/ModelPlan/Status/index.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+  Button,
+  Dropdown,
+  Grid,
+  GridContainer,
+  IconArrowBack,
+  Label
+} from '@trussworks/react-uswds';
+import { ErrorMessage, Field, Form, Formik, FormikProps } from 'formik';
+import * as Yup from 'yup';
+
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import FieldGroup from 'components/shared/FieldGroup';
+import modelStatus from 'constants/enums/modelPlanStatuses';
+import GetModelPlanQuery from 'queries/GetModelPlanQuery';
+import { GetModelPlan } from 'queries/types/GetModelPlan';
+import { UpdateModelPlan as UpdateModelPlanType } from 'queries/types/UpdateModelPlan';
+import UpdateModelPlan from 'queries/UpdateModelPlan';
+import { ModelStatus } from 'types/graphql-global-types';
+import { translateModelPlanStatus } from 'utils/modelPlan';
+
+type StatusFormProps = {
+  status: ModelStatus | undefined;
+};
+
+const Status = () => {
+  const { t } = useTranslation('modelPlan');
+  const { t: h } = useTranslation('draftModelPlan');
+  const { modelId } = useParams<{ modelId: string }>();
+
+  const history = useHistory();
+  const formikRef = useRef<FormikProps<StatusFormProps>>(null);
+  const validationSchema = Yup.object().shape({
+    status: Yup.string().required('Enter a role for this team member')
+  });
+
+  const { data } = useQuery<GetModelPlan>(GetModelPlanQuery, {
+    variables: {
+      id: modelId
+    }
+  });
+
+  const { status } = data?.modelPlan || {};
+
+  const [update] = useMutation<UpdateModelPlanType>(UpdateModelPlan);
+
+  const handleFormSubmit = (formikValues: StatusFormProps) => {
+    update({
+      variables: {
+        id: modelId,
+        changes: {
+          status: formikValues.status
+        }
+      }
+    })
+      .then(response => {
+        if (!response?.errors) {
+          history.push(`/models/${modelId}/task-list/`);
+        }
+      })
+      .catch(errors => {
+        formikRef?.current?.setErrors(errors);
+      });
+  };
+
+  return (
+    <MainContent>
+      <GridContainer>
+        <Grid col={6}>
```

Comment:
This should have a responsive breakpoint for tablet/mobile

`desktop={{ col: 6 }}`